<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlyphTestWindow</class>
 <widget class="QMainWindow" name="GlyphTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="PHierTab">
       <attribute name="title">
        <string>PHierTab</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="phier_view_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="phier_func_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Candidate Selection(Sampling)</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>System Sampling</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>X:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="x_number_edit"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                       <item>
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>Y:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="y_number_edit"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QPushButton" name="sys_sampling_button">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Execute</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Random Sampling</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>N:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="n_number_edit"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="random_sampling_button">
                   <property name="text">
                    <string>Execute</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Layout Optimization</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string>Data Cost</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <item>
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>Weight:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Proximity:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>70</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Similarity:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <item>
                      <widget class="QLineEdit" name="data_weight_edit">
                       <property name="text">
                        <string>5000</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="data_proximity_edit">
                       <property name="text">
                        <string>0.5</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="data_similarity_edit">
                       <property name="text">
                        <string>3</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_8">
                <property name="title">
                 <string>Smoothness Cost</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QLabel" name="label_10">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>70</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Weight:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="smoothness_weight_edit">
                     <property name="text">
                      <string>2000</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>Label Cost</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                     <item>
                      <widget class="QLabel" name="label_9">
                       <property name="minimumSize">
                        <size>
                         <width>70</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Weight:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>Proximity:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Similarity</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <item>
                      <widget class="QLineEdit" name="label_weight_edit">
                       <property name="text">
                        <string>1000</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="label_proximity_edit">
                       <property name="text">
                        <string>0.5</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="label_similarity_edit">
                       <property name="text">
                        <string>0.5</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="exec_singopt_button">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Execute</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>368</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="HierTab">
       <attribute name="title">
        <string>HierTab</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
